Sources/SectionScrubber.swift

```diff
@@ -194,14 +192,14 @@ public class SectionScrubber: UIView {
 
             let yPosInContentInContentView = calculateYPosInContentView(forYPosInView: newYPosForSectionScrubber)
 
-            self.delegate?.sectionScrubber(self, didRequestToSetContentViewToYPosition: yPosInContentInContentView)
+            self.collectionView.setContentOffset(CGPoint(x: 0, y: yPosInContentInContentView), animated: false)
```

Comment:
Is that why it won't respect boundaries and allow you to scroll past its bounds?